webapp/templates/public/team_submissions.html.twig

```diff
@@ -0,0 +1,30 @@
+{% extends "public/base.html.twig" %}
+
+{% block title %}
+    {% if team is not empty %}Submissions for team {{ team.effectiveName }} on problem {{ problem.problem.name }} - {% endif %}{{ parent() }}
+{% endblock %}
+
+{% block content %}
+    <h1 class="mt-3">
+        Submissions for team {{ team.effectiveName }} on problem {{ problem | problemBadge }} {{ problem.problem.name }}
```

Comment:
Why do we need to test for `team` in the `title block` but not in the content block?

Reply:
This twig is only used for non static, non modal (if you right click -> open in new tab) pages, so we know the team.